src/hooks/v2v3/transactor/DistributePayouts.ts

```diff
@@ -8,29 +13,40 @@ import { TransactorInstance } from 'hooks/Transactor'
 import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
 import { useContext } from 'react'
 import { useV2ProjectTitle } from '../ProjectTitle'
+import { ETH_PAYMENT_TERMINAL_V_3_1 } from '../V2V3ProjectContracts/projectContractLoaders/ProjectPrimaryEthTerminal'
 
-type DistributePayoutsTx = TransactorInstance<{
-  memo?: string
+interface DistributePayoutsTxBaseParams {
   amount: BigNumber | undefined
   currency: V2V3CurrencyOption | undefined
-}>
+}
+
+type DistributePayoutsTx3Params = DistributePayoutsTxBaseParams & {
+  memo?: string
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
should we just disable this eslint rule? i feel like it's used in multiple places, or could be

Reply:
yeah good call